lib/action.ml

```diff
@@ -155,9 +155,28 @@ module Action (Github_api : Api.Github) (Slack_api : Api.Slack) = struct
         )
       )
 
-  let generate_notifications (ctx : Context.t) req =
+  let ignore_notifications_from_user cfg req =
+    let sender_login =
+      match req with
+      | Github.Push n -> Some n.sender.login
```

Comment:
wait what? it now ignores all commits that are pushed by this user? That was not part of the plan now, smells security issue to me, please only ignore non-code changes (ie comments)

Reply:
@Khady said we should fine-grain it later, so should I create another issue to fine-grain it? Or should we not allow the silencing of all notifications as a feature? but yes I agree we should be notified if some bot manages to push code--possible security issue

Reply:
quick fix is to remove the line

Reply:
which ones should we keep? my initial thinking is keeping only this: `| Issue_comment n -> Some n.sender.login` the rest we don't ignore (we will be notified if the user does anything but comment on PR)